lte/gateway/python/scripts/config_stateless_agw.py

```diff
@@ -19,6 +19,7 @@
 import os
```

Comment:
Add info at the top of this file for why we're keeping this as a separate file -- if I recall, it's because it's intended to be run directly by systemd?